lib/api/src/sys/externals/memory.rs

```diff
@@ -60,6 +60,13 @@ impl Memory {
         })
     }
 
+    /// Create a memory object from an existing memory and attaches it to the store
+    pub fn new_from_existing(new_store: &mut impl AsStoreMut, memory: VMMemory) -> Self {
```

Comment:
Can this be `Into<VMMemory>`? That way, any implementation that implements the `LinearMemory` trait will work

Reply:
but about the store parameter?